src/service/asset-api-adapter.js

```diff
@@ -0,0 +1,248 @@
+ * group name and userId are optional and will use the logged user's group and userId if needed
+ *
+ *       var aa = new F.service.Asset({
+ *          account: 'acme-simulations',
+ *          project: 'supply-chain-game',
+ *          group: 'team1',
+ *          userId: ''
+ *       });
+ *       aa.create('test.txt', {
+            encoding: 'BASE_64',
+            data: 'VGhpcyBpcyBhIHRlc3QgZmlsZS4=',
+            contentType: 'text/plain'
+ *       }, { scope: 'user' })
+ */
+
+'use strict';
+
+var ConfigService = require('./configuration-service');
+var StorageFactory = require('../store/store-factory');
+// var qutil = require('../util/query-util');
+var TransportFactory = require('../transport/http-transport-factory');
+var _pick = require('../util/object-util')._pick;
+var keyNames = require('../managers/key-names');
+
+var apiEndpoint = 'asset';
+
+module.exports = function (config) {
+    var store = new StorageFactory({ synchronous: true });
+    var session = JSON.parse(store.get(keyNames.EPI_SESSION_KEY) || '{}');
+    var defaults = {
+        /**
+         * For projects that require authentication, pass in the user access token (defaults to empty string). If the user is already logged in to Epicenter, the user access token is already set in a cookie and automatically loaded from there. (See [more background on access tokens](../../../project_access/)).
+         * @see [Authentication API Service](../auth-api-service/) for getting tokens.
+         * @type {String}
+         */
+        token: store.get(keyNames.EPI_COOKIE_KEY) || '',
+        /**
+         * The project id. If left undefined, taken from the URL.
+         * @type {String}
+         */
+        account: undefined,
+        /**
+         * The account id. In the Epicenter UI, this is the **Team ID** (for team projects). If left undefined, taken from the URL.
+         * @type {String}
+         */
+        project: undefined,
+        /**
+         * The group name. Defaults to session's groupName.
+         * @type {String}
+         */
+        group: session.groupName,
+        /**
+         * The group name. Defaults to session's userId.
+         * @type {String}
+         */
+        userId: session.userId,
+        /**
+         * The API's scope. Valid values are: user, group and project.
+         * @see [Asset API documentation](https://forio.com/epicenter/docs/public/rest_apis/asset/) for the required permissions to write to each scope.
+         * @type {String}
+         */
+        scope: 'user',
+
+        fullUrl: true,
+
+        /**
+         * The transport are the options passed to the XHR request.
+         * Defaults the contentType to 'multipart/form-data' as is the most common way to upload a file, through a input[type=file].
+         * @type {object}
+         */
+        transport: {
+            contentType: false,
```

Comment:
does "false" here mean that you use "multipart/form-data", or is the comment out of date? 
if I set this myself, should I use a boolean or a string?